kubernetes/cmsweb/services/couchdb.yaml

```diff
@@ -67,11 +67,11 @@ spec:
               - sudo chmod 0777 /data/srv/state/couchdb/database; sudo chown _couchdb:_couchdb /data/srv/state/couchdb/database
         resources:
           requests:
-            memory: "256Mi"
-            cpu: "500m"
+            memory: "512Mi"
+            cpu: "1000m"
           limits:
             memory: "3Gi"
-            cpu: "2000m"
+            cpu: "4000m"
```

Comment:
Do we use couchdb in k8s setup anymore? I thought that we separated them and put couchdb node separate from k8s. If so, there is no need to change anything here.

Reply:
Yes, but in case we decide to deploy it again in k8s, we at least have a better hardware description.